📝 Update readme (#6)
indigotech/onboard-henrique-matheus: README.md

# TaqFood
TaqFood is an app for odering food directly from your phone that offers a several options of restaurants
## Enviroment and Tools

> This aplication was designed using React-Native with a Typescript layout. Here are some tools it uses:

Reviewer: ```suggestion
This application was built using React-Native and Typescript. Here are some other tools it uses:
```
